deliverables/homework.playground/Contents.swift

```diff
@@ -3,16 +3,82 @@
 import Foundation
 
 //: Create an empty array to store shopping list items
+var shoppingList = [String]()
 
 //: Add at least 5 items to the array using 3 different insertion methods
+shoppingList.insert("eggs", atIndex: 0)
+shoppingList.append("milk")
+shoppingList +=  ["lettuce"]
+shoppingList += ["hamburger"]
+shoppingList.insert("ketchup", atIndex: 3)
+shoppingList.append("bread")
 
 //: Sort your array of items
+shoppingList = shoppingList.sort()
```

Comment:
shoppingList.sort { (a, b) -> Bool in
    a < b
}
shoppingList.sortInPlace()
shoppingList = shoppingList.sort(>)
shoppingList.sort(>)
shoppingList.sort({ $0.compare($1) == .OrderedAscending})